CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG for 2.x
 =================
 
+* 2.4.2 (xxxx-xx-xx)
+
+    * Revert automatic cancellation of pending collection promises once the
+      output promise resolves. This was introduced in 42d86b7 and was both
```

Comment:
42d86b7 should probably be replaced with a version tag and/or PR as these are more accessible.